CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Reporting a bug
+## What is a bug?
+Bugs are defects in the software that are not working as expected. This can also be performance issues.
+## How to report bugs?
+First of all ensure that the bug was not already reported by searching on GitHub under Issues.
```

Comment:
Das

Reply:
Das gilt für alle issue types also bitte in der neuformulierung das für alle berücksichtigen.